prboom2/src/CMakeLists.txt

```diff
@@ -517,50 +517,48 @@ function(AddGameExecutable TARGET SOURCES)
         ${CMAKE_BINARY_DIR}
         ${CMAKE_CURRENT_SOURCE_DIR}
     )
+
     target_link_libraries(${TARGET} PRIVATE
         ${SDL2_LIBRARIES}
+        SDL2_mixer::SDL2_mixer
+        ZLIB::ZLIB
```

Comment:
Is the change here because the old syntax indicated it was optional, but it should be here since it's required?

Reply:
The old syntax guarded zlib behind `if(ZLIB_FOUND)`, which is no longer necessary since zlib is declared as required (`find_package(ZLIB REQUIRED)`).

Since I had to edit the calls to `target_link_libraries` anyway to change `*_LIBRARIES` into the target, I just moved all the required targets together.

Reply:
👍